crates/parser/tests/happy_path_test.rs

```diff
@@ -95,6 +95,9 @@ fn assert_file_is_parsed_with_config(file: PathBuf, mut config: ParseConfig) {
     config
         .add_values
         .insert(JsonPointer::from(canary_ptr), json!(true));
+    config
```

Comment:
FYI: I added this so that the filename will get included in the error message if it fails to parse.